universe/Fleet.h

```diff
@@ -46,6 +46,11 @@ FO_ENUM(
     ((NUM_FLEET_AGGRESSIONS))
 )
 
+namespace FleetDefaults {
+    constexpr auto FLEET_DEFAULT_ARMED = FleetAggression::FLEET_AGGRESSIVE;
```

Comment:
this is just so you can put the alias in all the places where a new fleet is created and change the default consistently by changing the single value here?

Reply:
That's the idea. Plus it is more verbose to use an explicit default value.